### PR TITLE
Upgrade core libs

### PR DESCRIPTION
**Pull request by j4qfrost**
This allows coverage using lcov and grcov. Related to this issue https://github.com/servo/core-foundation-rs/pull/357

### CODE REVIEW

**Comment by jdm** [comment]
This can merge after #181.

File: harfbuzz-sys/Cargo.toml [hunk 1]
```diff
@@ -27,8 +27,8 @@ pkg-config = { version = "0.3", optional = true }
 cc = { version = "1", optional = true }
 
 [target.'cfg(target_vendor = "apple")'.dependencies]
-core-graphics = "0.17"
-core-text = "13"
+core-graphics = ">=0.19"
+core-text = ">=15"
```
Comment (jdm) [inline harfbuzz-sys/Cargo.toml:31]: No need for the >=s. This can be `"0.19"` and `"15"` with the same effect.

Comment (j4qfrost) [reply harfbuzz-sys/Cargo.toml]: Done.

**Pull request merged**